integrations/helm/release/release.go

```diff
@@ -112,11 +112,29 @@ func (r *Release) GetUpgradableRelease(name string) (*hapi_release.Release, erro
 	}
 }
 
+func (r *Release) mustRollback(name string) (bool, error) {
```

Comment:
I would call it `shoulRollback` since, by convention, `must` is used as a prefix in Golang for something which will panic if doesn't succeed.